dll/win32/shell32/lang/bg-BG.rc

```diff
@@ -622,7 +622,7 @@ BEGIN
     PUSHBUTTON "Обзор...", IDC_BUTTON_PATH, 135, 17, 50, 14
     EDITTEXT IDC_EDIT_PATH, 7, 17, 123, 14, ES_AUTOHSCROLL
     LTEXT "Select an icon from the list below:", -1, 7, 36, 179, 10
-    LISTBOX IDC_PICKICON_LIST, 7, 47, 181, 120, LBS_OWNERDRAWFIXED | LBS_NOINTEGRALHEIGHT | LBS_MULTICOLUMN | LBS_DISABLENOSCROLL | WS_HSCROLL | WS_TABSTOP
+    LISTBOX IDC_PICKICON_LIST, 7, 47, 181, 120, LBS_OWNERDRAWFIXED | LBS_NOINTEGRALHEIGHT | LBS_MULTICOLUMN | LBS_DISABLENOSCROLL | LBS_HASSTRINGS | WS_HSCROLL | WS_TABSTOP
```

Comment:
Can you explain to me why LBS_HASSTRINGS is needed?

Reply:
@HBelusca 

This ListBox has LBS_OWNERDRAWFIXED style, so LB_ADDSTRING, LB_GETITEMDATA, LB_SETITEMDATA, and LB_FINDSTRINGEXACT messages have different behavior.

https://msdn.microsoft.com/en-us/library/windows/desktop/bb775181.aspx

> If the list box has an owner-drawn style but not the LBS_HASSTRINGS style, this parameter is stored as item data instead of a string.

https://msdn.microsoft.com/en-us/library/windows/desktop/bb761346.aspx

> If the item is in an owner-drawn list box created without the LBS_HASSTRINGS style, this message replaces the value contained in the lParam parameter of the LB_ADDSTRING or LB_INSERTSTRING message that added the item to the list box.

Reply:
We use these strings to associate between the icon indeces and RT_GROUP_ICON resource IDs.

Reply:
We use these strings to associate between the icon indeces and RT_GROUP_ICON resource IDs.

Reply:
lParam cannot be used for that purpose? Or is it already used for something else?

Reply:
No, it can't. An RT_GROUP_ICON entry can have its string-named ID.

Reply:
lParam is already used for icon handles.

Reply:
Current:
phIcons: (icon index) → (icon handle).
LParam: (icon index) → (icon handle).
Search by String: (RT_GROUP_ICON ID) → (icon index)

Reply:
You suggested:
phIcons: (icon index) → (icon handle).
Search by lParam: (RT_GROUP_ICON ID) → (icon index).
OK. It's possible.

Reply:
Did without LBS_HASSTRINGS style. Done.